pom.xml

```diff
@@ -207,6 +207,32 @@
     <dependencyManagement>
         <dependencies>
             <dependency>
+                <groupId>org.apache.httpcomponents</groupId>
```

Comment:
Are all these dependencies provided by core AEM bundles? Im not sure that they are...
Ex. My AEM 6.0 cannot find locate a bundle that exports `org.apache.http.client.fluent.Request` via depfinder..

The jackson stuff seems to be exported via a AEM Social UGC bundle (TBH the way soco iterates i wouldn't be surprised if that export disappeared in a few versions; almost feels like spillage.)